bluehex.html

```diff
@@ -0,0 +1,263 @@
+<html>
+<head>
+<style>
+.energy {
+  float: left;
+}
+.player {
+  background: #d5bdd2;
+}
+#wrapper {
+  height: 700px;
+  width: 700px;
+  background: #f6f6f4;
+}
+.box {
+  width:70px;
+  height:70px;
+  border:1px solid white;
+  float: left;
+}
+@media only screen and (max-width:1000px) {
+  #wrapper {
+    height: 600px;
+    width: 600px;
+  }
+  .box {
+    width:60px;
+    height:60px;
+    border:1px solid white;
+    float: left;
+  }
+}
+@media only screen and (max-width:710px) {
+  #wrapper {
+    height: 500px;
+    width: 500px;
+  }
+  .box {
+    width:50px;
+    height:50px;
+    border:1px solid white;
+    float: left;
+  }
+}
+.block {
+  background: #1e1e1e;
+}
+.remove {
+  background: white;
+}
+.board {
+  float: left;
+}
+.text {
+  float: left;
+  font-family: 'Amatic SC';
+}
+</style>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
+integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css"
+href="https://fonts.googleapis.com/css?family=Pacifico">
+<link rel="stylesheet" type="text/css"
+href="https://fonts.googleapis.com/css?family=Amatic+SC">
+
+</head>
+<body>
+  <div class = "container">
+    <div class = "row">
+      <center><div style="font-family: 'Pacifico', serif; font-size: 60px;">Complete the day</div>
```

Comment:
You should try not to use inline css. Assign it a class or ID and style this instead. 

```css
text-align: center;
```

is how you center things without the `<center>` tag